### PR TITLE
OPTIMIZE-IMPORTS Remove unused imports

### PR DESCRIPTION
**Pull request by votemike**
Removing unused imports

### CODE REVIEW

File: tests/TwigEnvironmentProvider.php [hunk 1]
```diff
@@ -2,8 +2,8 @@
 declare(strict_types = 1);
 namespace Tests\App;
 
-use App\Ds2013\PresenterFactory as Ds2013PresenterFactory;
 use App\Ds2013\PresenterFactory;
```
Comment (votemike) [inline tests/TwigEnvironmentProvider.php]: Should this even be here?

Comment (BjornTwachtmann) [reply tests/TwigEnvironmentProvider.php]: Please don't change this. It's there for a reason https://github.com/bbc/programmes-frontend/blob/eb31694fa34e8ac8f279caad378a5cd7bf43df68/tests/TwigEnvironmentProvider.php#L120 . That reason is not my personal amusement.

Comment (votemike) [reply tests/TwigEnvironmentProvider.php]: Grim.
Is it OK to re-order it? Intellij also re-order when it cleans up unused imports.

Comment (votemike) [reply tests/TwigEnvironmentProvider.php]: Wait, your line doesn't even use this line. It uses the one above/below it.

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: tests/TwigEnvironmentProvider.php (hunk 1 above)
Comment (BjornTwachtmann) [reply tests/TwigEnvironmentProvider.php]: use App\Ds2013\PresenterFactory as Ds2013PresenterFactory;
...
  self::$ds2013PresenterFactory = new Ds2013PresenterFactory($translateProvider, $router, $helperFactory); 

Unless I'm being more dense than usual that import is needed. If you want to refactor it to use the fully qualified class name or something do so, I'm completely unconcerned either way.

Comment (votemike) [reply tests/TwigEnvironmentProvider.php]: My comment is on the `use App\Ds2013\PresenterFactory;` line. I can't see a `PresenterFactory;` being used in this file.

Comment (BjornTwachtmann) [reply tests/TwigEnvironmentProvider.php]: Ah, okay, I was somewhat confused by the diff showing the used import being removed. The other one probably isn't needed.

Comment (votemike) [reply tests/TwigEnvironmentProvider.php]: The diff should be clearer now...

**Comment by votemike** [comment]
Test this please

**Pull request merged**